### PR TITLE
Minor : Fix #18606 prevent overwriting style for PUT request

### LINKED ISSUE #18606: styling (color, icon) is lost if a glossaryTerm is updated via the bulk upload?
**Affected module**
UI

**Describe the bug**
If make changes to the glossaryTerm via upload to lose any applied styling (color, icon).

**Expected behavior**
**Or maintain style via import
Or don't change if uploaded via import**

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 1.5.4]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

### PR DESCRIPTION
**Pull request by sonika-shah**
### Describe your changes:

Fixes #18606

I worked on ... because ...

#
### Type of change:

- [x] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/EntityRepository.java [hunk 1]
```diff
@@ -2924,6 +2925,14 @@ private static List<EntityReference> getEntityReferences(List<EntityReference> r
 
     private void updateStyle() {
       if (supportsStyle) {
+        Style originalStyle = original.getStyle();
+        Style updatedStyle = updated.getStyle();
+
+        if (originalStyle == updatedStyle) return;
+        if (operation == Operation.PUT && updatedStyle == null) {
```
Comment (harshach) [inline openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/EntityRepository.java:2932]: Lets not add these kind of conditions. PUT as definition requires the full payload.If someone is using PUT and. not passing all the required details then we will replace it.
What use-case are we trying to solve here

Comment (sonika-shah) [reply openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/EntityRepository.java:2932]: @harshach  while importing glossary csv the existing style of the glossary terms are getting removed, 

while  importing we either create or update entity, while updating entity internally a PUT request is being sent , and it overrides the field for which it does not find value in `updated` entity.

Comment (harshach) [reply openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/EntityRepository.java:2932]: @sonika-shah that shouldn't be the reason to add these kind of chcecks to allow specific thing to work.
The PR should be to load the attributes of glossaryTerm first and override with the import ones  but not allow PUT to override certain behavior

**Comment by harshach** [comment]
@sonika-shah lets revert to the previous version. That looks better

**Pull request merged**